lib/notifications/display.dart

```diff
@@ -89,13 +90,15 @@ class NotificationDisplayManager {
 
   static void _onMessageFcmMessage(MessageFcmMessage data, Map<String, dynamic> dataJson) {
     assert(debugLog('notif message content: ${data.content}'));
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
     final title = switch (data.recipient) {
       FcmMessageStreamRecipient(:var streamName?, :var topic) =>
         '$streamName > $topic',
       FcmMessageStreamRecipient(:var topic) =>
         '(unknown stream) > $topic', // TODO get stream name from data
       FcmMessageDmRecipient(:var allRecipientIds) when allRecipientIds.length > 2 =>
-        '${data.senderFullName} to you and ${allRecipientIds.length - 2} others', // TODO(i18n), also plural; TODO use others' names, from data
+        zulipLocalizations.notifGroupDmConversationLabel(
```

Comment:
nit in commit message: summary line should be kept on one line:

    notif: Internationalize group DM notification conversation
    label
    
    This commit addresses the issue of handling plural and
    […]

See https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-messages .

Just wrapping the same text onto a single line does it:
`notif: Internationalize group DM notification conversation label`
as that's only 64 columns wide. Commit summary lines can go up to a bit over 70 columns.

But we can improve the summary line further by tightening some: "notification" is covered by the prefix, and "internationalize" is a long word that has… well, the related term "internationalization" has a standard abbreviation. So:
`notif: Add i18n for conversation label on group DMs`

Reply:
Fixed.